test/modellib/ModelLib.cpp

```diff
@@ -120,6 +121,7 @@ bool ModelLibBuilder::checkSharedLibInstruction(
   }
   return true;
 }
+#endif
```

Comment:
Checking whether a symbol exists in a shared library can be done more easily with `dlopen` and `dlsym`.

Reply:
Done. Thanks!